Source/Core/Core/HW/WiimoteReal/IONix.cpp

```diff
@@ -84,9 +83,15 @@ void WiimoteScanner::FindWiimotes(std::vector<Wiimote*> & found_wiimotes, Wiimot
 	inquiry_info scan_infos[max_infos] = {};
 	auto* scan_infos_ptr = scan_infos;
 	found_board = nullptr;
+	// query in limited IAC - liac - using code from hcitool
+	// utils/tools/hcitool.c
+	const int liac_value = 0x9e8b00; // LIAC (https://www.bluetooth.org/en-us/specification/assigned-numbers/baseband)
```

Comment:
(Actually I'd also s/uint8_t/u8/ for consistency with Dolphin's usage of sized integer types -- ours predate C++11 and are still present pretty much all over the codebase.)